### PR TITLE
feat: images with different size

### PR DESCRIPTION
**Pull request by amarphule**
Images with different sizes
1. Thumbnail images
2. Circular/Round images
3. Responsive images
4. Align images

### CODE REVIEW

**Comment by nikhiljugale007** [comment]
for image round you can use aspect-ration:1/1 property.
You can also use object fit property to fit image properly in parent div.

File: components/images.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.img {
+  max-width: 50rem;
```
Comment (prathmeshjagtap) [inline components/images.css]: you can add object-fit : contain /cover to make it more precise and responsive everything  else look fine

**Review by prathmeshjagtap — APPROVED** [review]

File: components/images.css (hunk 1 above)
Comment (amarphule) [reply components/images.css]: Thank you. done with object-fit property

**Comment by amarphule** [comment]
> for image round you can use aspect-ration:1/1 property. You can also use object fit property to fit image properly in parent div.

Thank you for suggestion.

File: components/images.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+.img-thumbnail {
+  max-width: 12rem;
+  max-height: 12rem;
+  border-radius: 4px;
+}
+.img-round {
+  width: 25rem;
+  border-radius: 50%;
+  height: 25rem;
+}
```
Comment (AmishMishra11) [inline components/images.css:14]: I'd suggest making this responsive too.

**Review by AmishMishra11 — COMMENTED** [review]
good class naming, clean code. just work on responsiveness

File: components/images.css (hunk 2 above)
Comment (amarphule) [reply components/images.css:14]: thank you, I'll do that

**Comment by amarphule** [comment]
fix: suggested changes

File: components/images.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  object-fit: contain;
+}
+.img-thumbnail {
+  max-width: 12rem;
+  max-height: 12rem;
+  border-radius: 4px;
+}
+.img-round {
+  width: 25rem;
+  border-radius: 50%;
```
Comment (nikhiljugale007) [inline components/images.css:12]: for image round you can use aspect-ration:1/1 property.
You can also use object fit property to fit image properly in parent div.

**Review by nikhiljugale007 — COMMENTED** [review]
Added one suggestion.
Rest look good

**Pull request merged**